lightningd/channel.h

```diff
@@ -306,6 +306,9 @@ struct channel {
 
 	/* Do we allow the peer to set any fee it wants? */
 	bool ignore_fee_limits;
+
+	/* Set to `true` if we have received a channel_reestablish in future. */
+	bool lost_state;
```

Comment:
You don't need to add this: channel->future_per_commitment_point is non-NULL in this case (also, you forgot to restore it on restart anyway, so not adding a field is easier!).

Reply:
But there's no field in the output `listpeerchannels` which has it.

Reply:
Thanks for pointing out.